packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -20,47 +20,47 @@ export class VaButton {
   /**
    * If `true`, the button will use `Back` as its text and an icon to represent going back in form flows.
    */
-  @Prop() back?: boolean = false;
+  @Prop({ reflect: true }) back?: boolean = false;
 
   /**
    * If `true`, the button will use the big variant.
    */
-  @Prop() big?: boolean = false;
+  @Prop({ reflect: true }) big?: boolean = false;
 
   /**
    * If `true`, the button will use `Continue` as its text and an icon to represent proceeding forward in form flows.
    */
-  @Prop() continue?: boolean = false;
+  @Prop({ reflect: true }) continue?: boolean = false;
 
   /**
    * If `true`, the component-library-analytics event is disabled.
    */
-  @Prop() disableAnalytics?: boolean = false;
+  @Prop({ reflect: true }) disableAnalytics?: boolean = false;
 
   /**
    * If `true`, the click event will not fire.
    */
-  @Prop() disabled?: boolean = false;
+  @Prop({ reflect: true }) disabled?: boolean = false;
 
   /**
    * The aria-label of the component.
    */
-  @Prop() label?: string; // could use this.el.getAttribute('aria-label') but this is more explicit
+  @Prop({ reflect: true }) label?: string; // could use this.el.getAttribute('aria-label') but this is more explicit
 
   /**
    * If `true`, the button will use the secondary variant.
    */
-  @Prop() secondary?: boolean = false;
+  @Prop({ reflect: true }) secondary?: boolean = false;
 
   /**
    * If `true`, the button will submit form data when clicked.
    */
-  @Prop() submit?: boolean = false;
+  @Prop({ reflect: true }) submit?: boolean = false;
 
   /**
    * The text displayed on the button. If `continue` or `back` is true, the value of text is ignored.
    */
-  @Prop() text?: string;
+  @Prop({ reflect: true }) text?: string;
```

Comment:
Why do we need all of these props reflected?

Reply:
I've updated it to only reflect props that are needed for styling. Without reflecting props like `continue`, styling rules such as `:host([continue]:not([continue='false']))` don't get applied because the prop isn't reflected in the DOM.